### PR TITLE
feat: trash page

### PR DESCRIPTION
**Pull request by tusharkandpal**
To check:

- `trash-context.js`
- various services for `trash`

**Note:** Can ignore `Note.jsx`, `Nav.jsx` & other component changes. Also, backend folder & `server.js`.

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
The code structure looks really good. Really liked the way you have well structured the context files and services related to shifting the notes between the archive and trash sections!!
https://deploy-preview-11--digi-notes.netlify.app/
The live preview of Digi Notes is also working out. Good work @tusharkandpal !!

File: src/context/trash-context.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+  useEffect(() => {
+    if (isLoggedIn) {
+      (async () => {
+        const { data, status } = await getTrashService();
+        if (status === 200) {
+          setTrash(data.trash);
+        }
+      })();
+    } else setTrash([]);
```
Comment (JantuDeb) [inline src/context/trash-context.js:29]: initial value is empty `[]` no need to do it again

**Review by JantuDeb — APPROVED** [review]
The code looks really good,

File: src/context/trash-context.js (hunk 1 above)
Comment (tusharkandpal) [reply src/context/trash-context.js:29]: it's for when user log out & login again.

**Pull request merged**